test/e2e/etcd_compaction_test.go

```diff
@@ -43,32 +42,11 @@ var _ = Describe("Etcd Compaction", func() {
 			provider := p
 			Context(fmt.Sprintf("with provider %s", provider.Name), func() {
 				BeforeEach(func() {
-					etcdName = fmt.Sprintf("etcd-%s", provider.Name)
+					etcdName = fmt.Sprintf("etcd-compaction-%s", provider.Name)
+					storePrefix = etcdName
 					storageContainer = getEnvAndExpectNoError(envStorageContainer)
 
-					By("Purge snapstore")
-					snapstoreProvider := provider.Storage.Provider
-					if snapstoreProvider == druidstore.Local {
-						purgeLocalSnapstoreJob := purgeLocalSnapstore(parentCtx, cl, storageContainer)
-						defer cleanUpTestHelperJob(parentCtx, cl, purgeLocalSnapstoreJob.Name)
```

Comment:
Is this change required to make tests run concurrently? If no, then please revert back this change.

Reply:
PTAL https://github.com/gardener/etcd-druid/pull/833/files/9207d33b14a8f2324756d60ab407722880eae629#r1713247574